app/services/threat_level/assessment_runs_data_builder_service.rb

```diff
@@ -0,0 +1,18 @@
+module ThreatLevel
+  class AssessmentRunsDataBuilderService
```

Comment:
Seems like a very convoluted Java-esque way of doing things. Is this class really necessary? :)